src/components/dialogs/load-modification-dialog.js

```diff
@@ -127,7 +126,11 @@ const LoadModificationDialog = ({
         formProps: filledTextField,
         enumValues: LOAD_TYPES,
         defaultValue: formValues?.loadType ? formValues.loadType.value : '',
-        previousValue: loadInfos?.type,
+        previousValue: loadInfos?.type ? (
```

Comment:
Maybe move this change in `useEnumValue` hook then it will be applied for all previousValue of an Enum

Reply:
fixed, existed already but wasn't used correctly here